youtube_dl/extractor/yourporn.py

```diff
@@ -33,11 +32,13 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(url, video_id)
 
-        video_url = urljoin(url, self._parse_json(
-            self._search_regex(
-                r'data-vnfo=(["\'])(?P<data>{.+?})\1', webpage, 'data info',
-                group='data'),
-            video_id)[video_id]).replace('/cdn/', '/cdn5/')
+        url_parts = self._parse_json(self._search_regex(
+            r'data-vnfo=(["\'])(?P<data>{.+?})\1', webpage, 'data info', group='data'), video_id)[video_id].split("/")
+
+        aid = self._search_regex(r'data-aid=\'(.*?)\'', webpage, 'aid')
```

Comment:
Must allow both quotes kind. Do not capture empty string. Again read coding conventions.